benchmarks/django_simple/meta.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "django-simple",
+    "run": "gunicorn config.wsgi --bind 0.0.0.0:5000 --chdir=/app --pid gunicorn.pid",
+    "iterations": 3,
+    "variants": {
+        "disabled": {},
+        "tracer": {
+            "run": "ddtrace-run gunicorn config.wsgi --bind 0.0.0.0:5000 --chdir=/app --pid gunicorn.pid"
+        },
+        "profiler": {
+            "env": {
+                "BENCHMARK_ONLY_PROFILING_ENABLED": "1"
```

Comment:
do we have the usage of this configured yet? I didn't see it in the sea of django configs 😆

Reply:
Where is that env variable used?

Reply:
Added this into the wsgi file so the profiler is started manually without `ddtrace-run`. This removes the tracer from the picture.